packages/nextjs/src/models.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Object model for the app router static params.
+ */
+export type StaticParams = {
```

Comment:
I would suggest to move it under "/client/models.ts", since it's used only there